### ISSUE TITLE
Code cifar10 uses CPU instead of GPU

### ISSUE THREAD

**Issue by usernamemissing** [tensorflow/tensorflow#26476]
`
import time
time.sleep(1.5)
from keras.datasets import cifar10
from matplotlib import pyplot
from scipy.misc import toimage

import numpy
from keras.datasets import cifar10
from keras.models import Sequential
from keras.layers import Dense
from keras.layers import Dropout
from keras.layers import Flatten
from keras.constraints import maxnorm
from keras.optimizers import SGD
from keras.layers.convolutional import Conv2D
from keras.layers.convolutional import MaxPooling2D
from keras.utils import np_utils
from keras import backend as K
K.set_image_dim_ordering('th')

seed = 7
numpy.random.seed(seed)

(X_train, y_train), (X_test, y_test) = cifar10.load_data()

X_train = X_train.astype('float32')
X_test = X_test.astype('float32')
X_train = X_train / 255.0
X_test = X_test / 255.0

y_train = np_utils.to_categorical(y_train)
y_test = np_utils.to_categorical(y_test)
num_classes = y_test.shape[1]

model = Sequential()
model.add(Conv2D(32, (3, 3), input_shape=(3, 32, 32), padding='same', activation='relu', kernel_constraint=maxnorm(3)))
model.add(Dropout(0.2))
model.add(Conv2D(32, (3, 3), activation='relu', padding='same', kernel_constraint=maxnorm(3)))
model.add(MaxPooling2D(pool_size=(2, 2)))
model.add(Flatten())
model.add(Dense(512, activation='relu', kernel_constraint=maxnorm(3)))
model.add(Dropout(0.5))
model.add(Dense(num_classes, activation='softmax'))

epochs = 25
lrate = 0.01
decay = lrate/epochs
sgd = SGD(lr=lrate, momentum=0.9, decay=decay, nesterov=False)
model.compile(loss='categorical_crossentropy', optimizer=sgd, metrics=['accuracy'])
print(model.summary())

model.fit(X_train, y_train, validation_data=(X_test, y_test), epochs=epochs, batch_size=32)

scores = model.evaluate(X_test, y_test, verbose=0)
print("Accuracy: %.2f%%" % (scores[1]*100))`

**OUTPUT OF CODE**

Using TensorFlow backend.
2019-03-08 13:22:50.598025: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2019-03-08 13:22:50.617421: E tensorflow/stream_executor/cuda/cuda_driver.cc:300] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
2019-03-08 13:22:50.617482: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:163] retrieving CUDA diagnostic information for host: linuxbox-1
2019-03-08 13:22:50.617489: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:170] hostname: linuxbox-1
2019-03-08 13:22:50.617516: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:194] libcuda reported version is: 390.116.0
2019-03-08 13:22:50.617533: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:198] kernel reported version is: 390.77.0
2019-03-08 13:22:50.617539: E tensorflow/stream_executor/cuda/cuda_diagnostics.cc:308] kernel version 390.77.0 does not match DSO version 390.116.0 -- cannot find working devices in this configuration

can anyone help me with performance?@tensorflow

**Comment by jvishnuvardhan** [comment]
@usernamemissing Could you check running the following code to see whether TF is detecting GPU or not
from tensorflow.python.client import device_lib 
print(device_lib.list_local_devices())

What do you see as output? Please fill the requested details in the [template](https://github.com/tensorflow/tensorflow/issues/new?template=10-build-installation-issue.md). Without these details it is difficult to support. Thanks!

**Comment by grzegorz700** [comment]
I got the same output (`kernel version 390.77.0 does not match DSO version 390.116.0`). When I applied the last Ubuntu updates, and I restarted the system this message disappeared.

**Comment by jvishnuvardhan** [comment]
@usernamemissing Could you follow @grzegorz700 suggestion and let me know how it progresses. Please close the issue if it was resolved already. Thanks!

**Comment by jvishnuvardhan** [comment]
I think it was resolved. I am closing the issue. Please open a new ticket if you see similar issue again. Thanks!
